packages/frontend/app/pods/components/pod/application/off-canvas-container/sidebar/contacts/component.ts

```diff
@@ -25,12 +25,13 @@ export default class ContactsSidebar extends Component<IArgs> {
   }
 
   get contacts() {
+    const sortedContacts = this.allContacts.sort((a, b) =>
```

Comment:
Nested ternaries can be hard to read. I should really have a lint rule for them ;)
Anyway you can extract the sortComparator to a function?
It can live in module space above or below the class